FirstApiSandbox/Database/BookData.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using FirstApiSandbox.Model;
+
+namespace FirstApiSandbox.Database
+{
+    public class BookData
+    {
+        public static List<Book> bookList = new List<Book> {
+            new Book{ Name="EmptyList", Genre="NA", Author="NA" }
+        };
+
+        public List<Book> GetBooksFromDatabase()
+        {
+            return bookList;
+        }
+
+        public Book GetBooksFromDatabaseAtIndex(string name)
+        {
+            Debug.WriteLine(name);
+
+            var bookToBeReturned = bookList.Where(iteratorBook => iteratorBook.Name.Equals(name)).FirstOrDefault();
+            Debug.WriteLine(bookToBeReturned);
+            return bookToBeReturned;
+        }
+
+        public void AddBookInDatabase(Book newBook)
+        {
+            bookList.Add(newBook);
+        }
+
+        public bool UpdateBookInDatabase(string name, Book newBook)
```

Comment:
Your Namespace Is already {ProjectName}.Database and Your Class Is already Book
So Just Write Method Name As Update()